starters/features/auth/src/routes/plugin@auth.ts

```diff
@@ -3,13 +3,13 @@ import GitHub from '@auth/core/providers/github';
 import type { Provider } from '@auth/core/providers';
 
 export const { onRequest, useAuthSession, useAuthSignin, useAuthSignout } = serverAuth$(
-  ({ env }) => ({
-    secret: env.get('AUTH_SECRET'),
+  () => ({
+    secret: import.meta.env.VITE_AUTH_SECRET,
```

Comment:
i have changed this because whenever i am use the `env.get("VITE_AUTH_SECRET")` it is always giving `undefined` which is causing server config error.